bindings/python/build_defs.oss.bzl

```diff
@@ -77,6 +77,125 @@ def iree_py_test(**kwargs):
     #   imports
     py_test(legacy_create_init = False, **kwargs)
 
+def iree_py_test_suite(
```

Comment:
If you define these in a ".oss" file you're going to need to write internal analogs as well (we can discuss more the setup here). I think they should be portable though, so instead define them in a separate bzl file. We can workshop the name and location. We've got a weird mix of conventions for where bzl files go

Reply:
We're do we want these to go? It seemed weird to me that `iree_py_test` was hanging out in the bindings but was also used by the integrations, could you touch on that too?

Reply:
It has do with the python bindings specifically, which integrations that use python rely on. It only needs to exist because there are subtle differences between the bazel py rules in OSS and internal to Google so we need a shim to make them behave the same. I think they could easily go somewhere in `build_tools/bazel` though. But iree_py_test_suite as a wrapper doesn't need a separate internal version.

It could go in a bzl file just called `iree_py_test_suite.bzl` which could either be here or somewhere in build_tools. Let's see if someone else has an opinion? Maybe add Stella as a reviewer.

Reply:
Ah okay, that makes sense.

Reply:
For now I'm going to draft putting it in in `build_tools/bazel/iree_py_test_suite.bzl`.